### DISCUSSION TITLE
Feature Flags Client / Server

### DISCUSSION THREAD

**Discussion by tbd-develop** [tbd-friends/water-drink-water#54] [General]
What framework is in place for Feature Flags, do we "roll our own" or is there existing frameworks that can accommodate this.

**Comment by CodeItQuick** [comment]
There are existing frameworks, we'd have to look into them. I think we can just roll our own for the first bit, as its a fair bit of time investment to put an entire framework in.

**Comment by tbd-develop** [comment]
https://github.com/microsoft/FeatureManagement-Dotnet
